File: core/src/kr/edoli/imview/Textures.java
package kr.edoli.imview;

import com.badlogic.gdx.graphics.Color;
import com.badlogic.gdx.graphics.Pixmap;
import com.badlogic.gdx.graphics.Texture;

/**
 * Created by 석준 on 2016-02-06.
 */
public class Textures {
    public static Texture White = colorTexture(Color.WHITE);

    private static Texture colorTexture(Color color) {
        Pixmap pixmap = new Pixmap(1, 1, Pixmap.Format.RGBA8888);

        pixmap.setColor(color);
        pixmap.fill();

        Texture texture = new Texture(pixmap);

        pixmap.dispose();

        return texture;
    }
}

File: core/src/kr/edoli/imview/ui/Textures.kt
package kr.edoli.edoliui.widget

import com.badlogic.gdx.graphics.Color
import com.badlogic.gdx.graphics.Pixmap
import com.badlogic.gdx.graphics.Texture

/**
 * Created by daniel on 16. 1. 8.
 */
object Textures {
    val white = produceWhiteTexture()

    private fun produceWhiteTexture(): Texture {
        val pixmap = Pixmap(1, 1, Pixmap.Format.RGB565)

        pixmap.setColor(Color.WHITE)
        pixmap.fill()

        val tex = Texture(pixmap)
        tex.setFilter(Texture.TextureFilter.Linear, Texture.TextureFilter.Linear)

        pixmap.dispose()

        return tex
    }
}
